clients/drcachesim/scheduler/scheduler.h

```diff
@@ -1490,14 +1514,13 @@ template <typename RecordType, typename ReaderType> class scheduler_tmpl_t {
     // Opens up all the readers for each file in 'path' which may be a directory.
```

Comment:
I know it didn't change in this PR, but I think this comment could be clearer.
It doesn't mention that open_readers is selective about what it opens (subject to
only_threads and only_shards), and on my initial reading the first sentence seemed
to say it only opens readers for paths that might be directories.

How about something like "Opens readers for each file in 'path', subject to the
constraints in 'reader_info'.  'path' may be a directory."

Reply:
Done.  Similar for open_reader().  Also updated the Returns line to say Updates the ti2dinput, unfiltered_tids, and input_count fields of 'reader_info'..